pkg/resources/nodeagent/nodeagent.go

```diff
@@ -199,7 +199,7 @@ func NodeAgentFluentbitDefaults(userDefined **v1beta1.NodeAgent) (*v1beta1.NodeA
 var NodeAgentFluentbitWindowsDefaults = &v1beta1.NodeAgent{
 	FluentbitSpec: &v1beta1.NodeAgentFluentbit{
 		FilterKubernetes: v1beta1.FilterKubernetes{
-			KubeURL:       "https://kubernetes.default.svc.cluster.local:443",
```

Comment:
Should this patch be preserved?

Reply:
Or we can simply remove it everywhere like I proposed in https://github.com/banzaicloud/logging-operator/issues/1021#issuecomment-1195832840. WTYT @ahma @siliconbrain